multikueue-clusterprofile/README.md

```diff
@@ -0,0 +1,336 @@
+```
+
+4. Apply the Terraform configuration:
+
+```
+terraform apply
+```
+
+When prompted, enter `yes`, to confirm actions.
+
+This command may take a minute to complete.
+
+The output is similar to the following:
+
+```
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+```
+
+### Option 2: Manual setup with gcloud CLI
+
+This option uses gcloud CLI to manually create the clusters and configure MultiKueue.
+
+#### Deploy Infrastructure
+
+1. Enable the required Google Cloud APIs:
+```shell
+gcloud services enable \
+    cloudresourcemanager.googleapis.com \
+    compute.googleapis.com \
+    container.googleapis.com \
+    gkehub.googleapis.com \
+    connectgateway.googleapis.com
+```
+
+2. To create a worker cluster, run:
+```shell
+gcloud container clusters create-auto multikueue-worker \
+  --project=${PROJECT_ID} \
+  --region=${LOCATION} \
+  --enable-fleet
+```
+
+Install kueue in the cluster following the [documentation](https://kueue.sigs.k8s.io/docs/installation/#install-a-released-version).
+
+3. To create a hub cluster, run:
+```shell
+gcloud container clusters create multikueue-hub \
+  --project=${PROJECT_ID} \
+  --enable-fleet \
+  --region=${LOCATION} \
+  --workload-pool=${PROJECT_ID}.svc.id.goog \
+  --labels=fleet-clusterinventory-management-cluster=true,fleet-clusterinventory-namespace=kueue-system
+```
+
+Install kueue in the cluster following the [documentation](https://kueue.sigs.k8s.io/docs/installation/#install-a-released-version).
+
+4. Verify that `ClusterProfile` objects are generated in the hub cluster:
+```shell
+gcloud container clusters get-credentials multikueue-hub \
+  --location=${LOCATION} \
+  --project=${PROJECT_ID}
+
+kubectl get clusterprofile -n kueue-system
+```
+
+For more details, see the documentation for the [ClusterProfile sync feature][21].
+
+5. Grant the KSA the required IAM roles:
+```shell
+PROJECT_ID=`gcloud config get-value project`
+PROJECT_NUMBER=`gcloud projects describe "${PROJECT_ID}" --format "value(projectNumber)"`
+gcloud projects add-iam-policy-binding projects/${PROJECT_ID} --condition=None \
+--role=roles/gkehub.gatewayEditor \
+--member=principal://iam.googleapis.com/projects/${PROJECT_NUMBER}/locations/global/workloadIdentityPools/${PROJECT_ID?}.svc.id.goog/subject/ns/kueue-system/sa/kueue-controller-manager
+
+gcloud projects add-iam-policy-binding projects/${PROJECT_ID} --condition=None \
+--role=roles/container.developer \
+--member=principal://iam.googleapis.com/projects/${PROJECT_NUMBER}/locations/global/workloadIdentityPools/${PROJECT_ID?}.svc.id.goog/subject/ns/kueue-system/sa/kueue-controller-manager
+```
+
+#### Deploy MultiKueue
+
+Manually install and configure MultiKueue with ClusterProfile API following the [documentation](https://kueue.sigs.k8s.io/docs/tasks/manage/setup_multikueue/#setup-multikueue-with-clusterprofile-api). Remember to use the plugin image you built in the 'Build and Push the GCP Auth Plugin Image' section.
```

Comment:
The "Build and Push the GCP Auth Plugin Image" section is actually called "Prepare the plugin container image" from what I can see.

Also, how about adding an anchor link to that section? This should work:
```suggestion
Manually install and configure MultiKueue with ClusterProfile API following the [documentation](https://kueue.sigs.k8s.io/docs/tasks/manage/setup_multikueue/#setup-multikueue-with-clusterprofile-api). Remember to use the plugin you built in the [image preparation section](#prepare-the-plugin-container-image) section.
```

I don't have a strong opinion on the wording tbh, the text can also say "'Prepare the plugin container image' section" or anything you prefer.